src/include/sof/sof.h

```diff
@@ -54,6 +54,14 @@ struct sa;
 	__attribute__((unused))		\
 	typedef char META_CONCAT(assertion_failed_, MESSAGE)[(COND) ? 1 : -1]
 
+/* merges two 16-bit values into a single 32-bit value */
+#define merge_16b16b(high, low) (((uint32_t)(high) << 16) | \
```

Comment:
Good idea, can you put these in math/numbers.h as that's where there is similar code. Thanks

Reply:
Done